dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -826,18 +829,17 @@ public void publishRelatedHtmlPages(Contentlet contentlet) throws DotStateExcept
                 //Search for the page with a given identifier and for a given language (in case of Pages as content)
                 page = loadPageByIdentifier(htmlPageIdentifier.getId(), true, languageId, APILocator.getUserAPI().getSystemUser(), false);
 
-                if(page != null && page.isLive()){
+                if(null != page && page.isLive()){
                     //Rebuild the pages' files
                     PageServices.invalidateAll(page);
                 }
-            }
-            catch(Exception e){
-                String htmlPageIdentifierId = htmlPageIdentifier!=null?htmlPageIdentifier.getId():null;
-                String pageInode = page!=null?page.getInode():null;
-                Logger.error(this.getClass(), "Cannot publish related HTML Pages" +
-                        ". htmlPageIdentifier.getId(): " + htmlPageIdentifierId +
-                        ". LanguageId:" + languageId +
-                        ". pageInode:" + pageInode, e);
+            } catch(Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)

Reply:
Exception is not being logged on purpose.